### ISSUE TITLE
A question about training code in "pointcept/engines/train.py"

### ISSUE THREAD

**Issue by yangrongkun** [RayYoh/SGIFormer#6]
<img width="644" height="142" alt="Image" src="https://github.com/user-attachments/assets/c332b450-b861-4600-a5d2-c42973dd291a" />

Dear authors, I notice that the code in this repo doesn't use "self.scaler.unscale_(self.optimizer)" before clip the gradient and backward step, as shown in the figure above.

However, I found that the PointCept github repo contains this "self.scaler.unscale_(self.optimizer)" , does this code affect the training?

**Comment by RayYoh** [comment]
Hi @yangrongkun , thanks for pointing this out. Actually, I am not very sure whether it affects the performance. I see Pointcept add clip_grad in this commit ([link](https://github.com/Pointcept/Pointcept/commit/6df01eeb811bbbde9677a2449dab1bfabb5efaa3#:~:text=%2B-,self.scaler.unscale_(self.optimizer),)).

As Pointcept now also supports SGIFormer, I think you can try it.
